tests/crm/test_crm.py

```diff
@@ -248,6 +248,9 @@ def verify_thresholds(duthost, asichost, **kwargs):
                 # in order to test percentage threshold (Can't even reach 1 percent)
                 # For test case used 'nexthop_group' need to be configured at least 1 percent from available
                 continue
+            if "ipv4 neighbor" or "ipv6 neighbor" in kwargs["crm_cli_res"] and "cisco-8000" in duthost.facts["asic_type"].lower():
```

Comment:
"ipv4 neighbor" or "ipv6 neighbor" in kwargs["crm_cli_res"] will always to be "ipv4 neighbor". Use kwargs["crm_cli_res"] in ["ipv4 neighbor", "ipv6 neighbor"] instead.